chroma_core/models/host.py

```diff
@@ -46,33 +46,14 @@
 from chroma_core.services.job_scheduler import job_scheduler_notify
 from iml_common.lib.util import ExceptionThrowingThread
 from chroma_core.models.sparse_model import VariantDescriptor
+from django.contrib.postgres.aggregates import BoolOr
```

Comment:
I might be missing it, but how is this being used?

Reply:
It's being used here:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/52bc9f87b7c23a840f2a2a6129ecb7e577dad340/chroma_core/models/host.py#L392